bsnes/ruby/video/qtopengl.cpp

```diff
@@ -52,6 +52,8 @@ class pVideoQtOpenGL {
     void paintGL() {
       unsigned outputWidth  = width();
       unsigned outputHeight = height();
+      outputWidth *= QApplication::desktop()->devicePixelRatio();
+      outputHeight *= QApplication::desktop()->devicePixelRatio();
```

Comment:
I think this should be changed to something akin to what this snippet of code did for wider compatibility of Qt versions:
https://github.com/devinacker/bsnes-plus/blob/17818367f6832269006989de600e64cda0a16bf4/bsnes/ui-qt/debugger/ppu/image-grid-widget.cpp#L176-L182

Reply:
Oh nice didn't know there was a float version. I'll upgrade to this.

Reply:
Alright, upgraded to use this more portable code. Seems to work fine on my MacBook retina screen but I don't have any other high-dpi hardware to test with.